docs/developer/checkout.mdx

```diff
@@ -1657,3 +1657,139 @@ Anonymous and unfinished checkouts are deleted after a specified period of time
 - anonymous checkouts (neither user or email is set) after 30 days,
 - user checkouts (either user or email is set) after 90 days,
 - checkouts without lines after 6 hours.
+
+## Creating checkout from order
+
+:::info
+
+This feature was introduced in **Saleor 3.14**.
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+Saleor allows you to create a checkout from existing orders. This feature can be used when implementing the reordering functionality.
```

Comment:
What happens if the prices of the variants change? We should describe that here.

Reply:
We don't copy the prices. As prices can be reduced by sale that doesn't exists etc. I will mention this here.